### ISSUE TITLE
JSON Hyper-Schema draft-07 support?

### ISSUE THREAD

**Issue by handrews** [encode/jsonhyperschema-codec#12]
Hello,
  [JSON Hyper-Schema draft-07](http://json-schema.org/latest/json-schema-hypermedia.html) (technically [draft-handrews-json-schema-hyperschema-01](https://tools.ietf.org/html/draft-handrews-json-schema-hyperschema-01), but the meta-schema is numbered draft-07) was published recently.  Note that there are [minor errata](https://github.com/json-schema-org/json-schema-spec/issues?q=is%3Aissue+is%3Aopen+label%3Aerrata+label%3Ahypermedia).

  This is [a significant revamp of draft-04 hyper-schema](http://json-schema.org/draft-07/README.html).  Since draft-04 hyper-schema was never as broadly adopted as draft-04 validation, we felt free to make some major changes through drafts 05, 06, and 07.  Draft-07 is believed to be reasonably feature complete, and we do not anticipate major changes until we have feedback from implementations.  I can give more detail on what changes might occur.  Draft-08 will mostly be updates to the core and validation specs, as draft-07 was primarily an update to hyper-schema.

  Is this project interested in updating?  I am trying to figure out which projects are still active and working on support for recent drafts and which basically went on hold in the very long gap between drafts 04 and 05, and are no longer likely to be updated.

**Comment by lovelydinosaur** [comment]
Yeah I’ll likely be working on support for that at some point, although possible that with some rejigging it might not be against this repo in the end.

**Comment by handrews** [comment]
Cool, I'll keep the project on the active list, PR against https://github.com/json-schema-org/json-schema-org.github.io welcome for either updated draft support or a new repo!

So far the only notable idea floated for draft-08 hyper-schema is that maybe `headerSchema` should be called `targetMetaDataSchema`, and `targetHints` should be `targetMetaDataHints` (json-schema-org/json-schema-spec#566).  And the annotation collection process clarifications in json-schema-org/json-schema-spec#530 might slightly tweak how links are collected.  But there are no behavioral changes under consideration right now.
